### DISCUSSION TITLE
What could be causing my text to be invisible after typing?

### DISCUSSION THREAD

**Discussion by Nikola-Milovic** [zellij-org/zellij#2659] [Q&A]
Hello everyone, since this is the most active community, not sure where else to ask. I am playing around with newer tooling like zellij and  [starship.rs](https://starship.rs/). My terminal emulator is [kitty](https://sw.kovidgoyal.net/kitty/) and my shell is bash. The issue I am experiencing is that after certain commands like `docker logs` my input becomes invisible 

https://github.com/zellij-org/zellij/assets/50072027/6acf0d9c-9be9-4e96-9d69-613493d42cf8

When there `wthf`, I tried to type `echo wtf`, so echo for some reason doesn't work.

I can close the pane that's experiencing the issue and reopen it to fix it, but still annoying. Any pointers in what could be causing this interaction is greatly appreciated, never had this issue prior to going with `zellij` and `starship.rs`

**Comment by Lockszmith-GH** [comment]
kitty is great, but it has a ton of settings and the defaults are not always correct for the 'modern' world.

For comparison's sake - Have you compared to other terminals?
If you are on Windows: What happens when you use Windows Terminal and ssh into the machine - does it behave the same?

What is the TERM variable in your shell?
How does kitty's terminal configuration look like?

If Windows Terminal isn't an option, try it a modern, cross-plaform one like [Tabby](https://github.com/Eugeny/tabby) and see if this still persissts.

**Reply by Nikola-Milovic** [reply]
I tried a lot of different things and shells, after further debugging it seems that a `docker logs -f ` would not print anything and then quitting it would lead to empty input, after disableing [atuin](https://github.com/atuinsh/atuin), it seems that its fixed. Just not sure how atuin is doing this

**Reply by Nikola-Milovic** [reply]
Yep, found the issue

https://github.com/atuinsh/atuin/issues/996

**Reply by Lockszmith-GH** [reply]
did you try the `reset` shell command (it resets the terminal - can help when some rogue ansi code wasn't disabled properly)

**Reply by Lockszmith-GH** [reply]
Cool porject this atuin, now I learned something new

**Reply by Nikola-Milovic** [reply]
Yeah reset works, but its annoying nonetheless

**Reply by Lockszmith-GH** [reply]
Wasn't suggesting a solution - just an understanding of the nature of the problem. Thanks for updating.
